en/docs/learn/try-device-flow.md

```diff
@@ -0,0 +1,217 @@
+            </td>
+            <td>
+                key_length
+            </td>
+        </tr>
+        <tr class="odd">
+            <td>
+                <p>Expiry time</p>
+            </td>
+            <td>
+                <p>The expiry time of the user code in milliseconds.</p>
+            </td>
+            <td>
+                expiry_time
+            </td>
+        </tr>
+        <tr class="even">
+            <td>
+                <p>Polling interval</p>
+            </td>
+            <td>
+                <p>The minimum amount of time in seconds that the client should wait between polling requests to the
+                 token endpoint in milliseconds.</p> 
+            </td>
+            <td>
+                polling_interval
+            </td>
+        </tr>
+        <tr class="odd">
+            <td>
+                <p>Key set</p>
+            </td>
+            <td>
+                <p>The set of characters that is used to generate the user code.</p>
+            </td>
+            <td>
+                key_set
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Configuring Device Authorization Grant properties during deployment
+
+All the above parameters can be configured at the server level through the `deployment.toml` file. A sample configuration 
+is shown below. 
+
+```
+[oauth.grant_type.device_code]
+key_length = 7
+expiry_time = 60000
+polling_interval = 5000
+key_set = "BCDFGHJKLMNPQRSTVWXYZbcdfghjklmnpqrstvwxyz23456789"
+```
+
+    
+1. Navigate to <IS_HOME>/bin and start the server by executing the following command on a terminal window.
+
+    ``` java tab="Linux/MacOS"
+    sh wso2server.sh
+    ```
+    
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+
+2. Access the [WSO2 Identity Server Management Console] (https://localhost:9443/carbon) and log in using your user
+ name and password.
+
+3. On the Main menu, click Service Providers > Add and add a new service provider.
+
+    ![register-service-provider](../assets/img/using-wso2-identity-server/register-service-provider.png)
+    
+    ![register-service-provider-name](../assets/img/using-wso2-identity-server/register-sp-name.jpg)
+
+4. Expand Inbound Authentication Configuration and then expand OAuth/OpenID Connect Configuration. Click Configure.
+
+    ![register-service-provider-oauth](../assets/img/using-wso2-identity-server/register-sp-oauth.jpg)
+
+5. Select `urn:ietf:params:oauth:grant-type:device_code` to enable using the device flow grant type. 
+
+    !!! info
+    Since these are public clients, ensure that the Allow authentication without the client secret option is checked.
+
+6. Next, click Update to save the service provider configurations. Note the generated OAuth client key and client
+ secret.
+
+    ![consumer-key-service-provider-oauth](../assets/img/using-wso2-identity-server/get-oauth-consumer-key.jpg)
+
+    !!! tip
+    When configuring with your device, configure the client ID as your OAuth client-key.
```

Comment:
Please check if this is aligned a tab spacing apart from the tip macro for it to appear properly.

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2737/commits/ecb385b6c012018e8a1e500140130e8eb3fc8f6b